packages/app-store/_components/AppCard.tsx

```diff
@@ -81,7 +81,7 @@ export default function AppCard({
             </p>
           </div>
           <div className="ml-auto flex items-center space-x-2">
-            {app.credentialOwner && (
+            {app.credentialOwner && !isPlatform && (
```

Comment:
Avatar can't be displayed for a platform user, hence needed to add this `isPlatform` check